Makefile

```diff
@@ -7,10 +7,11 @@ ifeq (, $(shell command -v ${ENGINE}))
 	RUN = ${ENGINE} run -v $$(pwd):/bulebule:Z -u $$(id -u):$$(id -g) bulebule
 endif
 
-default: src/main.elf
+default: src
```

Comment:
Why?

Reply:
Because only `src`'s Makefile knows when `main.elf` needs to be regenerated (i.e.: when any source code file is modified).